apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/functions/ParseContainerFromHeaders.java

```diff
@@ -36,11 +36,13 @@
 
    @Override
    public Container apply(HttpResponse from) {
+      String bytesUsed = from.getFirstHeaderOrNull(CONTAINER_BYTES_USED);
+	String objectCount = from.getFirstHeaderOrNull(CONTAINER_OBJECT_COUNT);
       Container c = 
       Container.builder()
             .name(name)
-            .bytesUsed(Long.parseLong(from.getFirstHeaderOrNull(CONTAINER_BYTES_USED)))
-            .objectCount(Long.parseLong(from.getFirstHeaderOrNull(CONTAINER_OBJECT_COUNT)))
+            .bytesUsed(bytesUsed==null?0:Long.parseLong(bytesUsed))
+            .objectCount(objectCount==null?0:Long.parseLong(objectCount))
```

Comment:
Our checkstyle checks might complain about this. Use spaces: `bytesUsed == null ? 0 : `